controls/os_spec.rb

```diff
@@ -282,3 +282,25 @@
     end
   end
 end
+
+control 'os-14' do
+  impact 1.0
+  title 'Check pam config - RedHat specific'
```

Comment:
All controls that we are adding to the base line need to be applicable and tested on all major linux distributions.

Reply:
Hi @chris-rock,
in dev-sec/ansible-collection-hardening there is currently no configuration of this for other distributions and I am not the Debian master that can create them. So I only change some code in dev-sec/ansible-collection-hardening and create the missing test in this repo.

Reply:
Our code for PAM configuration on any non-RHEL based distribution is severely lacking.
I would propose to open an issue for this and going on with only RHEL support here.